candies/cli/parsers/standard.py

```diff
@@ -76,15 +77,65 @@ def configure(parser: ArgumentParser, command: Command, prefix: str = '.'):
         {'.command': 'tool', '..command': 'install', 'g': 'something'}.
     """
 
+    def is_(x: inspect.Parameter, of: type) -> bool:
+        return isinstance(x, of) or x == of
+
+    def configure_arg(x: inspect.Parameter, config: Dict):
+        parser.add_argument(parameter.name,
+                            **config)
+
+    def configure_flag(x: inspect.Parameter, config: Dict):
+        # This actually makes an arg to behave like a flag,
+        # so one could call `--x` instead of `--x y`.
+        config['action'] = 'store_true'
+
+        if parameter.default is parameter.empty:
+            config['default'] = False
+        else:
+            config['default'] = bool(parameter.default)
+
+        short = getattr(parameter.annotation, 'short', None)
+        short = short if short is not None else parameter.name[0]
+
+        if short is '':
+            raise ValueError(f"Short name of the flag '{parameter.name}' "
+                             f"'{short}' "
+                             f"is an empty string")
```

Comment:
Missing `.` at the end of the sentence.

Reply:
Or is it intentional?

Reply:
I prefer not to put `.` at the end of the sentence in error messages, because the message may be used in many ways, for example, as a title. So, probably, it's better for the user to resolve the last punctuation.